vite.config.js

```diff
@@ -42,13 +42,22 @@ export default defineConfig(() => {
       port: 8080
     },
     test: {
-      // simulate DOM with happy-dom
-      environment: 'happy-dom',
+      // simulate DOM with jsdom
+      environment: 'jsdom',
+
       // enable jest-like global test APIs
       globals: true,
+
       setupFiles: ['./tests/setup.ts'],
+
       // enable threads to speed up test running
       threads: true,
+
+      // optionally disable threads to work around canvas module warnings
+      // but this causes more tests to fail
+      // ref: https://github.com/vitest-dev/vitest/issues/740
+      // threads: false,
```

Comment:
Maybe we can use this in the future? I can delete if you think I should.

Reply:
Disabling threads would cause the tests to be super slow. Not sure if we want that.

Reply:
It's not as bad as you think. The first run was with `threads: true` and the second run with `threads: false`:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/0d42e283-8406-4cdd-850a-749fff31de45)

But with threads disabled, we have 28 failed unit tests and I didn't want to get into that.

I think it's probably a good idea to leave this code comment in place in case a future dev wants to try to fix the canvas module warnings. I couldn't find a real solution to those.